onvm/onvm_mgr/Makefile

```diff
@@ -35,6 +35,9 @@
 # (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
+# set this to 1 if onvm is using flow table lookup for incoming packets
```

Comment:
`Set this to 0 to disable flow table lookup for incoming packets`
If the default is 1, we should describe the alternative